### DISCUSSION TITLE
Welcome to settings Discussions!

### DISCUSSION THREAD

**Discussion by Feichtmeier** [ubuntu-flutter-community/settings#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by shan-shaji** [comment]
Hey @Feichtmeier,  I have no prior experience in writing any ubuntu application, Is there any particular feature that I can help to implement.

**Reply by Feichtmeier** [reply]
HI @shan-shaji thanks for contributing!

If you have knowledge in flutter from android/iOS there shouldn't be much new knowledge needed for you.
We make use of gsettings.dart for readin/writing into the ubuntu settings.

All pages that need to be implemented are marked as issues with the feature-or-page-missing tag :) 
Feel free to grab any of those

**Reply by shan-shaji** [reply]
@Feichtmeier Ok cool

**Comment by shan-shaji** [comment]
Hey @Feichtmeier, I am getting the below error when compiling the app.

CMake Error at /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:545 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:733 (_pkg_check_modules_internal)
  CMakeLists.txt:45 (pkg_check_modules)

Do you have any idea why this is happening?

**Reply by Feichtmeier** [reply]
@mivoligo has the same error
Maybe because libhandy and 20.04
Currently it looks like we need 21.10 for libhandy-1-dev to be installed. We use libhandy to get the rounded bottom corners
I'll need to set up a 20.04 vm to see if there is any work around

**Reply by mivoligo** [reply]
I can confirm that reverting `/linux` folder to the commit before introducing libhandy fixed the error.

**Comment by atsen-dev** [comment]
Hi, there's any chance to have this app included in futures Ubuntu releases ?
Thanks

**Reply by Feichtmeier** [reply]
I would love this :) Even if I am a Ubuntu community member I am not on the Ubuntu desktop team.
When the app has all pages needed to be "feature complete" (plus all those already implented nice extras) we should surely ask the team

**Reply by atsen-dev** [reply]
Okay, hopefully !

For you, we have at how percent to be considered as "done"?

**Reply by Feichtmeier** [reply]
In the readme at the start there is a track of the open pages :) displays checkmark could now be also checked I guess thanks to you :)
